package org.dev.pys.exception.AuthExceptions;

public class InvalidCredentialsException extends RuntimeException {
    public InvalidCredentialsException() {
    }
}
